openmc/nuclide.py

```diff
@@ -93,3 +96,293 @@ def scattering(self, scattering):
             raise ValueError(msg)
 
         self._scattering = scattering
+
+    def plot_xs(self, types, divisor_types=None, temperature=294., axis=None,
+                Erange=(1.E-5, 20.E6), sab_name=None, cross_sections=None,
+                **kwargs):
+        """Creates a figure of continuous-energy cross sections for this
+        nuclide
```

Comment:
Can this fit on the previous line? It looks like it would still be less than 80 characters.